### PR TITLE
docs:  Change position of some Props's types

### PR DESCRIPTION
**Pull request by bumjoo**
## Description

Chanage README.md of Button

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Comment by JeffGuKang** [comment]
@bumjoo  Is there no necessary option as https://github.com/dooboolab/dooboo-ui/tree/master/main/Accordion?

**Comment by bumjoo** [comment]
I don’t understand what you mean. sorry. 
please explain details. 
I just changed document.

File: main/Button/README.md [hunk 1]
```diff
@@ -6,27 +6,27 @@
...
-| indicatorColor        |                       | string       | '#ffffff' |
-| activeOpacity         |                       | number       | 0.5       |
-| text                  |                       | string       |           |
-| onPress               |                       | func         |           |
-| touchableOpacityProps | TouchableOpacityProps |              |
-| hoverStyle            | ViewStyle             |              |
-| Accent                | ViewStyle             |              |
-| hoverTextStyle        | TextStyle             |              |
-| Secondary             | ViewStyle             |              |
+|                       | necessary | types                 | default   |
```
Comment (JeffGuKang) [inline main/Button/README.md]: `required` will be better than necessary.

**Comment by JeffGuKang** [comment]
> I don’t understand what you mean. sorry.
> please explain details.
> I just changed document.

Is there no required props to use the Button? The original `Button` component's `onPress` and `title` props are [required props](https://reactnative.dev/docs/button#onpress).

**Comment by bumjoo** [comment]
@JeffGuKang 

Yes, I know that `onPress` and `title` are required props. 
But they are not required props in Button component code in dooboo-ui.
(https://github.com/dooboolab/dooboo-ui/blob/master/main/Button/index.tsx)

so, I think this README.md should be modified after modifying the Button code.

**Pull request merged**